src/bbs2gh/Services/IBbsArchiveDownloader.cs

```diff
@@ -1,11 +1,11 @@
 using System.IO;
 using System.Threading.Tasks;
+using OctoshiftCLI.Extensions;
 
 namespace OctoshiftCLI.BbsToGithub.Services;
 
 public interface IBbsArchiveDownloader
 {
-    const string DEFAULT_BBS_SHARED_HOME_DIRECTORY = "/var/atlassian/application-data/bitbucket/shared";
     const string EXPORT_ARCHIVE_SOURCE_DIRECTORY = "data/migration/export";
```

Comment:
What are your thoughts on supporting a Samba location that isn't hard-coded to inspect a subdirectory of the "shared Atlassian" directory?  For example, the `export/` directory could be shared directly (instead of the root shared directory).

I'm certainly good on shipping this as-is and possibly adding support for this in a future issue :slightly_smiling_face:

Reply:
It's totally possible with my changes, all they need to do is to then set the optional `--bbs-home` directory. SMB is supposed to be used only in windows and by default the root directory is always shared for admins.